packages/react-components/react-accordion/src/components/AccordionHeader/useAccordionHeader.tsx

```diff
@@ -67,7 +67,7 @@ export const useAccordionHeader_unstable = (
     expandIcon: resolveShorthand(expandIcon, {
       required: true,
       defaultProps: {
-        children: <ChevronRightRegular transform={`rotate(${expandIconRotation})`} />,
+        children: <ChevronRightRegular style={{ transform: `rotate(${expandIconRotation}deg)` }} />,
```

Comment:
my only thought here is, how would someone else customize the expand icon, and re-use this rotation?

Reply:
The guidance to users adding styling in our docs is to use `makeStyles` and add it to the `className` so I think that approach would still work here

Reply:
are they able to alter the className based on the open/closed state? A demo would probably be helpful to show others how we expect them to do it. Rotate an icon, change + to -, 'open' to 'closed' etc